config_files/abell2744_config.yaml

```diff
@@ -0,0 +1,57 @@
+paths:
```

Comment:
Maybe throw these files in a v1 directory?

Reply:
Done!